src/test/java/io/meeds/qa/ui/stepDefinitions/TasksStepDefinition.java

```diff
@@ -1284,4 +1284,9 @@ public void switchToTASKSTab() {
   public void hoverOnProjectManagerIcon() {
     tasksSteps.hoverOnProjectManagerIcon();
   }
+
+  @When("I wait 1 second")
+  public void waitOneSeconds() throws InterruptedException {
+    tasksSteps.waitOneSeconds();
+  }
```

Comment:
This should be parametrable, see example here https://github.com/Meeds-io/meeds-qa-ui/blob/795a43610a5334f33607768ae8f478cab45eae38/src/test/java/io/meeds/qa/ui/steps/definition/GenericStepDefinitions.java#L54-L57

```suggestion
  @When("I wait 1 second")
  public void waitOneSeconds() throws InterruptedException {
    tasksSteps.waitOneSeconds();
  }
```